### PR TITLE
Allow setting environment for UserPermissionsModule

### PR DESCRIPTION
**Pull request by fraxachun**
Environment is then passed to AccessControlService to allow defining different permissions for different environments.

### CODE REVIEW

File: packages/api/cms-api/src/user-permissions/user-permissions.service.ts [hunk 1]
```diff
@@ -102,7 +102,7 @@ export class UserPermissionsService {
         });
         if (this.accessControlService.getPermissionsForUser) {
             if (user) {
-                let permissionsByRule = await this.accessControlService.getPermissionsForUser(user);
+                let permissionsByRule = await this.accessControlService.getPermissionsForUser(user, this.options.environment);
```
Comment (thomasdax98) [inline packages/api/cms-api/src/user-permissions/user-permissions.service.ts:105]: I think you forgot passing it to `getContentScopesForUser`?

File: .changeset/nasty-ligers-remember.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+"@comet/cms-api": minor
+---
+
+Allow setting environment for User Permissions Module
```
Comment (thomasdax98) [inline .changeset/nasty-ligers-remember.md:5]: Where do I get the environment from in the app? NODE_ENV is production for our applications in all deployed envs if I remember correctly

**Review by johnnyomair — COMMENTED** [review]
The access control service needs to be defined in the application anyway, why not inject the config there and handle the environment in the application alone?

File: .changeset/nasty-ligers-remember.md (hunk 2 above)
Comment (nsams) [reply .changeset/nasty-ligers-remember.md:5]: I also don't understand how this is planned to be used. The name `environment` is maybe too unspecific.

And I remember we talked about this, but can you please add in the description what problem this solves?

**Pull request closed without merging**